setup.sh

```diff
@@ -263,6 +263,7 @@ function usage()
   say '--help                     | -h    Display this usage message'
   say "--install                  | -i    Install $app_name"
   say '--install-kernel-dev-deps  | -k    Installs all necessary dependencies to build the kernel according to your distribution'
+  say "--full-instalation         | -F    Install $app_name and its kernel development dependencies"
```

Comment:
{}

Reply:
This was already covered in the last commit of this PR